scalafix-core/shared/src/main/scala/scalafix/lint/LintMessage.scala

```diff
@@ -16,6 +16,12 @@ final case class LintMessage(
     position: Position,
     category: LintCategory
 ) {
+
+  def id(owner: RuleName): String = {
```

Comment:
I just realized this is implemented in LintCategory.key https://github.com/scalacenter/scalafix/blob/master/scalafix-core/shared/src/main/scala/scalafix/lint/LintCategory.scala#L20